### DISCUSSION TITLE
Register a user account with an address

### DISCUSSION THREAD

**Discussion by e-zer0** [saleor/saleor#8625] [Q&A]
I notice the accountRegister mutation has a limited amount of fields (i.e., email and password).  Is there a way to add an address without having to call a second mutation.  Calling a second mutation would require two transactions in which one could fail for whatever reason.  I need to create user accounts that have addresses attached right from the point of creation.

**Comment by prometixX** [comment]
Have you found a solution for that? 
My idea was, I could save the user address details in metadata and write a plugin that waits for account updates.
If the user activates his account the plugin creates a address with the metadata and removes it.
